src/models/applicant/read.js

```diff
@@ -231,3 +231,22 @@ export const projectOwner = async (identityId, id) => {
     throw new PermissionError()
   }
 }
+
+export const projectsOwner = async (identityId, ids) => {
+  const flattedIds = `(${ids.map((id) => `'${id}'`).join(',')})`
+  try {
+    const { rows } = await app.db.query(
```

Comment:
why using raw ? it may cause sql injections as IDs are getting from client use sql and if you having problem with arrays we already had examples with ids `AND id=ANY(${ids})` use like this

Reply:
Nice tip i always woundered how are we going to handle it couldn't find anything related, 
Thanks 👍